blog/20250528-virtual-hackathon/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "GeoJupyter virtual hackathon 2025-05-28"
+description: |
+  A GeoJupyter virtual hackathon. Open to all!
+date: "2025-05-28"
+image: "/assets/images/hackathon.jpg"
+author:
+  - name: "The GeoJupyter community"
+categories:
+  - "Hackathons"
+tags: [hackathons]
+---
+
+# GeoJupyter virtual hackathon 2025-05-28
+
+Please add new agenda items under the `New agenda items` heading!
+
+- [Join us on Zoom](https://berkeley.zoom.us/j/92451699568)
+  - [What time is the meeting in my time zone?](https://dateful.com/convert/utc?t=3pm)
+- [Notes from previous hackathons](https://geojupyter.org/blog/#category=Hackathons)
+- [GeoJupyter](https://geojupyter.org) handy links:
+  - [GitHub org](https://github.com/geojupyter)
+  - [Community calendar](https://geojupyter.org/calendar.html)
+  - [Zulip chat](https://jupyter.zulipchat.com/#narrow/channel/471314-geojupyter)
+
+
+## Sign in!
+
+Your name / GitHub ID / affiliation
+
+* Matt Fisher / @mfisher87 / Schmidt DSE
+* Yao-Ting Yao / @YaoTingYao / Clark CGA
+* Jon Marokhovsky / @jmarokhovsky / Clark CGA
+* Arjun Verma / @arjxn-py / QuantStack
+
+
+## Agenda & notes
+
+### ⚡ (5 minutes) Lightning intros
+
+Tell us about you in 30 seconds or less!
+
+
+### 🌐 (5 minutes) Lightning demo
+
+What's new? Show & tell.
+Post on Zulip to request a show & tell slot;
+by default, QuantStack will demo awesome JupyterGIS progress each meeting!
+
+
+### 💡 (5 minutes) What will we hack on?
+
+* **What do you want to work on today**?
+  * For ideas, check out the [hackathon](https://github.com/geojupyter/jupytergis/labels/hackathon)
+    and [good first issue](https://github.com/geojupyter/jupytergis/labels/good%20first%20issue)
+    labels on the JupyterGIS project!
+* **Add your ideas to the “ideas” list below**.
+* **Add your favorite emoji or a `+` next to ideas you’re excited about**.
+  * Press the colon (:) key on your keyboard or navigate to “Insert > Emoji” in the menu bar to open the emoji browser.
+
+
+#### Ideas
+
+* Explore for refactor opportunities +++
+* Explore for rough edges ++
+* Make Relative imports more readable and writeable ([#263](https://github.com/geojupyter/jupytergis/issues/263)) +++
+* Rename "Single symbol" -> "Simple" ([#719](https://github.com/geojupyter/jupytergis/issues/719)) +
+* [436](https://github.com/geojupyter/jupytergis/issues/436) - Python API
+
+
+### 🪄 (all the minutes) Hack together!
+
+Form teams from the ideas generated in the step above!
+
+
+#### Breakout rooms
+
+* Lobby: Refactoring
+
+
+### 💬 (10 minutes) Share out
+
+Think about:
+What exciting things did you accomplish?
+What loose ends remain?
+Big questions? Big ideas?
+
+Please write for people who don’t have full context; link to related issues and documentation!
+
+* Share out 1
+* Share out 2
+* ...
```

Comment:
```suggestion
* Started on <https://github.com/geojupyter/jupytergis/issues/263>, but got stuck because we didn't understand the bundling setup for the `base` module.
* Reviewed <https://github.com/geojupyter/jupytergis/pull/714>
```